1_fetch/src/fetch_sites.R

```diff
@@ -15,37 +15,21 @@ fetch_sites_from_sf <- function(ind_file, sf_object, dates, stream_params) {
     sites_df_cd <- dataRetrieval::whatNWISdata(stateCd = cd, parameterCd = stream_params$stage, service = "uv")
     sites_df <- rbind(sites_df, sites_df_cd)
   }
-  sites_df <- dplyr::filter(sites_df, site_tp_cd == "ST")
-  sites_df <- dplyr::filter(sites_df, end_date >= as.Date(dates$end))
-  sites <- sites_df$site_no
-
-  # write the data file and the indicator file
-  data_file <- as_data_file(ind_file)
-  saveRDS(sites, data_file)
-  gd_put(ind_file, data_file)
-
-}
-
-#' Subset the site list to ones relevant for the storm
-#'
-#' @param ind_file character file name where the output should be saved
-#' @param all_sites data.frame of sites pulled for the bbox
-subset_sites <- function(ind_file, all_sites) {
 
-  all_sites <- readRDS(scipiper::sc_retrieve(all_sites))
+  # Filtering applied to every storm
+  sites_df <- sites_df %>%
+    # we only need stream sites
+    filter(site_tp_cd == "ST") %>%
+    # keeps only sites that have data since the start of the storm
+    # if a gage goes out during the storm, this filter would still capture that gage
+    filter(end_date >= as.Date(dates$start))
 
-  # subset the sites from the wide net cast to ones relevant to the storm
-  # subset criteria TBD
-  sites_info <- dataRetrieval::readNWISsite(siteNumbers = all_sites)
-
-  #############********* temporarily grab only a few sites *********#############
-  set.seed(10)
-  sites_info <- dplyr::sample_n(sites_info, 50)
-  #############*********
+  # return only the site numbers
+  sites <- sites_df$site_no
```

Comment:
Should we maybe convert this to an sf data.frame? 
`sf::st_as_sf(sites, coords = c("dec_long_va", "dec_lat_va"), crs = 4326)`

Reply:
Hmm I think it would be easier to do this after the sites are confirmed. So maybe as a processing step right after the custom sites function?